src/guest_memory.rs

```diff
@@ -408,6 +439,26 @@ impl<T: GuestMemory> Bytes<GuestAddress> for T {
         Ok(())
     }
 
+    /// # Examples
+    ///
+    /// * Read bytes from /dev/urandom
+    ///
+    /// ```
+    /// # use std::fs::File;
+    /// # use std::path::Path;
+    /// # #[cfg(feature = "backend-mmap")]
+    /// # fn test_read_random() -> Result<u32, ()> {
```

Comment:
You need to exclude this test from Windows builds.

Reply:
Done.